src/serverlogging.h

```diff
@@ -35,15 +35,23 @@
 /* Classes ********************************************************************/
 class CServerLogging
 {
+
+    CServerLogging();
+
 public:
-    CServerLogging() : bDoLogging ( false ), File ( DEFAULT_LOG_FILE_NAME ) {}
+    static CServerLogging& getInstance()
+    {
+        static CServerLogging theInstance;
```

Comment:
OK, so the variable name isn't visible to the outside world, but I'm not keen... `slStaticInstance` would probably follow naming style better.